app/vendors/vendor-dashboard/subscription/SubscriptionPricingTable.tsx

```
@@ -0,0 +1,243 @@
+      (currentPlan === VendorTier.FREE &&
+        (tierId === VendorTier.STANDARD || tierId === VendorTier.PREMIUM)) ||
+      (currentPlan === VendorTier.STANDARD && tierId === VendorTier.PREMIUM)
+    ) {
+      setUpgradingTier(tierId);
+
+      try {
+        // For the purposes of this implementation, we'll just show a mock checkout
+        // This would be replaced with a real Stripe checkout integration
+
+        // Create a mock cart item for Stripe checkout
+        const cartItems = [
```

ESLint

[error] 'cartItems' is assigned a value but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)